### DISCUSSION TITLE
wiki/development/build-instructions/linux/

### DISCUSSION THREAD

**Discussion by giscus** [PrismLauncher/prismlauncher.org#677] [Wiki]
# wiki/development/build-instructions/linux/

An Open Source Minecraft launcher with the ability to manage multiple instances, accounts and mods. Focused on user freedom and free redistributability.

https://prismlauncher.org/wiki/development/build-instructions/linux/

**Comment by MinePossu** [comment]
libqt6networkauth6-dev missing from dependencies
